src/main/webapp/index.html

```diff
@@ -16,6 +17,9 @@
       <div id="form-title">
         <p>Copy your Tweet here</p>
       </div>
+      <label><input type="checkbox" onclick="setLocationCookie()" /> Use Location</label>
```

Comment:
[Nit] Input tag should be outside the label.

Reply:
This is the only way I am able to get the label and checkbox to appear inline with each other on the website. Let me know if you know another way.

Reply:
You can use the for attribute https://www.w3schools.com/tags/tag_label.asp

Reply:
That's still not working

Reply:
That's weird